.drone.yml

```diff
@@ -30,8 +30,13 @@ steps:
     environment:
       # Identifies false positives like missing 'selector'. On Kustomize is common to don't set them
       VALIDATE_KUBERNETES_KUBEVAL: "false"
-      # We use a lot of copy paste in our code, is intended. So it important to don't use this linter 
+      # We use a lot of copy paste in our code, is intended. So it important to don't use this linter
       VALIDATE_JSCPD: "false"
+      VALIDATE_YAML: "false"
+      VALIDATE_TERRAFORM_TFLINT: "false"
```

Comment:
are these 2 really necessary to turn off?

Reply:
Those are the same settings that I used in [this PR](sighupio/fury-kubernetes-monitoring#96).

Reply:
I was wondering because in the other modules we did not have the need to disable them. if there are some particular files that we consider correct but fail the linting we can always exclude them.

Disabling the YAML linter seems a little overkill to me because the module is mostly YAML 😄

Reply:
Let me double check

Reply:
Turns out disabling YAML validation is not needed 🤦